### ISSUE TITLE
make dev fail ERROR: Package 'dp-accounting' requires a different Python: 2.7.13 not in '>=3.6'

### ISSUE THREAD

**Issue by wenhuizhang** [OpenMined/PipelineDP#244]
## Description
make dev fail due to 

## How to Reproduce
```
wenhui.zhang@n231-227-105:~/PipelineDP$ make dev
pip install -r requirements.dev.txt
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maint
Defaulting to user installation because normal site-packages is not writeable
Collecting git+https://github.com/google/differential-privacy.git@327972c1ae710e8cd0a4754fffdd78c3500272ee#subdirectory=python (from 
  Cloning https://github.com/google/differential-privacy.git (to revision 327972c1ae710e8cd0a4754fffdd78c3500272ee) to /tmp/pip-req-b
  Running command git clone -q https://github.com/google/differential-privacy.git /tmp/pip-req-build-n0wb3A
  Running command git rev-parse -q --verify 'sha^327972c1ae710e8cd0a4754fffdd78c3500272ee'
  Running command git fetch -q https://github.com/google/differential-privacy.git 327972c1ae710e8cd0a4754fffdd78c3500272ee
  Running command git checkout -q 327972c1ae710e8cd0a4754fffdd78c3500272ee
ERROR: Package 'dp-accounting' requires a different Python: 2.7.13 not in '>=3.6'
Makefile:24: recipe for target 'dev' failed
make: *** [dev] Error 1

```

## Expected Behavior
ERROR: Package 'dp-accounting' requires a different Python: 2.7.13 not in '>=3.6'

## System Information
Linux n231-227-105 5.10.35.bsk.2-amd64 #5.10.35.bsk.2 SMP Debian 5.10.35.bsk.2 

wenhui.zhang@n231-227-105:~/PipelineDP$ python --version
Python 2.7.13
wenhui.zhang@n231-227-105:~/PipelineDP$ python3 --version
Python 3.11.0a5

wenhui.zhang@n231-227-105:~/PipelineDP$ pip3 --version
pip 21.2.4 from /usr/local/lib/python3.11/site-packages/pip (python 3.11)
wenhui.zhang@n231-227-105:~/PipelineDP$ pip --version
pip 20.3.4 from /usr/local/lib/python2.7/dist-packages/pip (python 2.7)

**Comment by chinmayshah99** [comment]
This is an issue with the python version. The pip installation is using python 2.7 while `pipeline-dp` as well as `dp-accounting` are available only for python 3.6+.

**Comment by wenhuizhang** [comment]
Issue resolved by using virtual env
